docs/topics/mta-web-viewing-an-assessment-report.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-reviewing-an-analysis-report_{context}"]
+= Viewing an assessment report
+
+An {ProductShortName} assessment report displays the aggregated assessement of the data obtained from multiple questionnaires for multiple applications.
+
+.Procedure
+
+. To view an assessment report, in *Migration* view, in the menu on the left side of the window, click *Report*.
++
+The aggregated assessment report of all aplications is displayed.
++
+image::mta-assessment-report.png[Assessment report]
++
+* To display a report on the data from a particular questionnaire, select the required questionnaire from a drop-down list of all questionnaires in the *Current landscape* pane of the report. By default, all questionnaires are selected.
+* In the *Identified risks* pane of the report, you can sort the displayed list by application name, level of risk, questionnaire, questionnaire section, question, and answer.
+* To display a report for a specific application:
+.. Click the link in the *Applications* column.
++
+The applications included in the link are displayed as a list.
+.. Click the required application.
++
+A side drawer opens.
++
+** To see the assessed risk level for the appllication, open the *Reports* tab.
```

Comment:
Application risk is displayed in **Details** Tab
![Screenshot from 2024-01-30 20-56-42](https://github.com/windup/windup-documentation/assets/87791967/8c4c237c-3cd2-4b5f-9667-3be51e1d4673)